name/static/name/js/stats.js

```diff
@@ -0,0 +1,159 @@
+(function() {
+  'use strict';
```

Comment:
It appears that this only makes `strict mode` available in the anonymous function. We should enforce it throughout the file, or remove it all together.